inputs/manager-inputs.yaml

```diff
@@ -29,6 +29,14 @@ inputs:
     type: boolean
     default: false
 
+  local_rest_cert_file:
```

Comment:
change local_rest_cert_file to **agent**_rest_cert_**path** - indicate it's the agent cert, and consistent suffix